src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -67,11 +67,11 @@ public function collect(Request $request, Response $response, \Exception $except
             'env' => isset($this->kernel) ? $this->kernel->getEnvironment() : 'n/a',
             'debug' => isset($this->kernel) ? $this->kernel->isDebug() : 'n/a',
             'php_version' => PHP_VERSION,
+            'php_architecture' => PHP_INT_SIZE * 8,
+            'php_locale' => \Locale::getDefault() ?: 'n/a',
+            'php_timezone' => date_default_timezone_get(),
             'xdebug_enabled' => extension_loaded('xdebug'),
-            'eaccel_enabled' => extension_loaded('eaccelerator') && ini_get('eaccelerator.enable'),
             'apc_enabled' => extension_loaded('apc') && ini_get('apc.enabled'),
```

Comment:
We could even drop the case of APC here, because APC does not have its opcode caching features anymore on PHP 5.5+ (as it can only be installed thanks to the BC layer of APCu)